test/solver/TestCGSolver.cpp

```diff
@@ -36,14 +47,75 @@ int main(int argc, char* argv[]) {
                 if (argv[2][0] == 'w') {
                     // If weak scaling is specified, increase the problem size
                     // along the Y axis such that each rank has the same workload
-                    // (simplest enlargement method)
+                    // (the simplest enlargement method)
                     ptY = 1 << (5 + (int)N);
                     pt  = 32;
                     info << "Performing weak scaling" << endl;
                     isWeak = true;
+                } else {
+                    if (argv[2][0] == 'j') {
```

Comment:
Is a switch better here? Then you can have a default case which throws an exception for unrecognized options.

Reply:
I think the whole input should be rewritten in order to fit more the style of the alpine command line input...

Reply:
Okay, I see